pybossa/cache/task_browse_helpers.py

```diff
@@ -302,6 +292,30 @@ def parse_tasks_browse_args(args):
 
     return parsed_args
 
+def parse_tasks_browse_order_by_args(order_by, display_info_columns):
+    order_by_result = None
+    order_by_dict = dict()
+
+    if order_by:
+        order_by_result = order_by.strip()
+
+        # allowing custom user added task.info columns to be sortable
+        allowed_sort_fields = allowed_fields.copy()
+        allowed_sort_fields.update({col: "task.info->>'{}'".format(col) for col in display_info_columns})
+        for clause in order_by.split(','):
```

Comment:
Looks like we don't support sorting by multiple columns. Why do you need this loop for splitting order_by comma?

Reply:
I suppose the intention was to maintain the existing code as much as possible. This code was pre-existing and moved inside this function.

Reply:
Added support for multiple columns via https://github.com/bloomberg/pybossa/pull/719/commits/86662a466a6796004c725b0e3356e38d2059c86f